sea-orm-cli/Cargo.toml

```diff
@@ -22,20 +22,26 @@ path = "src/lib.rs"
 [[bin]]
 name = "sea-orm-cli"
 path = "src/bin/main.rs"
-required-features = ["codegen"]
+required-features = ["async_std", "codegen"]
 
 [[bin]]
 name = "sea"
 path = "src/bin/sea.rs"
-required-features = ["codegen"]
+required-features = ["async_std", "codegen"]
 
 [dependencies]
 clap = { version = "^3.2", features = ["env", "derive"] }
 dotenv = { version = "^0.15" }
-async-std = { version = "^1.9", features = [ "attributes", "tokio1" ] }
+async-std = { version = "^1.9", features = [
+    "attributes",
+    "tokio1",
+], optional = true }
```

Comment:
I think `dotenv` is only used in `bin/main.rs` and `bin/sea.rs`. I guess we can enable both under the `default` feature? User can always disable the default features and then only enable the features they want.